Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "21a65a0eae9dbcf464e3df5f8b14f8b49cb966eb1bfa61e3ccf9a9e63085945a"
+            "sha256": "effb38045a509709aef13afe50f1022ab64a9855d99a958199709545b2095151"
```

Comment:
@sappelhoff Would you mind updating the pipenv on `master`, so we can clean this out of the PR with a merge?

Reply:
sure, see: https://github.com/bids-standard/bids-specification/pull/274